common/update_kubos.c

```diff
@@ -201,6 +201,7 @@ int update_kubos(bool upgrade)
     char * file;
     char * env_addr;
     char * dfu_info;
+    char * devstring;
```

Comment:
does this need to be char devstring[4], as well?

Reply:
Hm I wonder if this was marked between when we discussed this change offline and commit https://github.com/ASU-cubesat/sparcs-uboot/pull/1/commits/81d74cbf184dd986125bb672f67029d8ef7d396d

this is exactly the line that is changed in that commit.